uint16 buff_mask
